src/niri.rs

```diff
@@ -591,13 +591,15 @@ impl State {
             Backend::Tty(tty)
         };
 
+        let mod_key = get_mod_key(&*config.borrow(), &backend);
```

Comment:
Doesn't seem like you need to pass this in? Both config and backend are passed in, so you can get it inside.